README.md

```diff
@@ -68,64 +68,64 @@ available as
 # Data Entities
 
 Within the FIWARE platform, an entity represents the state of a physical or
-conceptural object which exists in the real world.
+conceptual object which exists in the real world.
 
 ## Entities within a stock management system
 
-Within our simple stock management system, currently have four types of entity.
-The relationship between our entities is defined as shown:
+Within our simple stock management system, we currently have four entity types.
+The relationships among our entities are defined as shown below:
```

Comment:
For some reason "among" felt more natural to me because I thought of entities as a group. But the sentence is tied to a list which picks out the actual entities so I agree with you, “between” is the right choice here. Thanks!!!